templates/zerver/emails/password_reset.html

```diff
@@ -15,3 +15,10 @@
     Your friends at Zulip HQ
 </p>
 {% endblock %}
+
+{% block manage_preferences %}
```

Comment:
I feel like we don't want/need a manage preferences block on the password reset email since it is a one-off email specifically requested by the user.  I did a quick search through my inbox and there wasn't an unsubscribe link in any of the password reset emails I've gotten lately.